cmd/explorer/main.go

```diff
@@ -0,0 +1,150 @@
+package main
```

Comment:
I don't think this code is related to this PR 🤔

Reply:
:see_no_evil: